src/Module/Tos.php

```diff
@@ -44,15 +65,15 @@ public static function content() {
 		$tpl = get_markup_template('tos.tpl');
 		if (Config::get('system', 'tosdisplay'))
 		{
-		return replace_macros($tpl, [
-			'$title' => L10n::t("Terms of Service"),
-			'$tostext' => BBCode::convert(Config::get('system', 'tostext')),
-			'$displayprivstatement' => Config::get('system', 'tosprivstatement'),
-			'$privstatementtitle' => L10n::t("Privacy Statement"),
-			'$privoperate' => L10n::t('At the time of registration, and for providing communications between the user account and their contacts, the user has to provide a display name (pen name), an username (nickname) and a working email address. The names will be accessible on the profile page of the account by any visitor of the page, even if other profile details are not displayed. The email address will only be used to send the user notifications about interactions, but wont be visibly displayed. The listing of an account in the node\'s user directory or the global user directory is optional and can be controlled in the user settings, it is not necessary for communication.'),
-			'$privdistribute' => L10n::t('This data is required for communication and is passed on to the nodes of the communication partners. Users can enter additional private data that may be transmitted to the communication partners accounts.'),
-			'$privdelete' => L10n::t('At any point in time a logged in user can export their account data from the <a href="%1$s/settings/uexport">account settings</a>. If the user wants to delete their account they can do so at <a href="%1$s/removeme">%1$s/removeme</a>. The deletion of the account will be permanent.', System::baseurl())
-		]);
+			return replace_macros($tpl, [
+				'$title' => L10n::t("Terms of Service"),
+				'$tostext' => BBCode::convert(Config::get('system', 'tostext')),
+				'$displayprivstatement' => Config::get('system', 'tosprivstatement'),
+				'$privstatementtitle' => L10n::t("Privacy Statement"),
+				'$privoperate' => L10n::t('At the time of registration, and for providing communications between the user account and their contacts, the user has to provide a display name (pen name), an username (nickname) and a working email address. The names will be accessible on the profile page of the account by any visitor of the page, even if other profile details are not displayed. The email address will only be used to send the user notifications about interactions, but wont be visibly displayed. The listing of an account in the node\'s user directory or the global user directory is optional and can be controlled in the user settings, it is not necessary for communication.'),
+				'$privdistribute' => L10n::t('This data is required for communication and is passed on to the nodes of the communication partners and stored there. Users can enter additional private data that may be transmitted to the communication partners accounts.'),
```

Comment:
Please make sure the translation string change is effective in the class member as well.